evmjit/libevmjit/Arith256.cpp

```diff
@@ -81,10 +81,12 @@ llvm::Function* generateLongMulFunc(char const* _funcName, llvm::IntegerType* _t
 	func->setDoesNotAccessMemory();
 	func->setDoesNotThrow();
 
-	auto x = &func->getArgumentList().front();
-	x->setName("x");
-	auto y = x->getNextNode();
-	y->setName("y");
+	auto iter = func->getArgumentList().begin();
+	llvm::Argument& x = (*iter++);
```

Comment:
If you want to use parentheses, I would find `*(iter++)` more readable, and the parentheses below can be dropped, I think.